considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md

```diff
@@ -0,0 +1,66 @@
+# Economic CAB Minutes
+
+## Meeting Information
+
+**Location:** Google Meet
+
+**Materials**: [Sip-015 proposal](https://github.com/stacksgov/sips/blob/feat/sip-015/sips/sip-015/sip-015-network-upgrade.md)
+
+**Date:** Oct 29 2022
+
+**Time:** 1pm ET
+
+**Attendees:**
+
+- mattyTokenomics
+- Chiente
+- Xan
+- Pieter
+
+100% attendance
+
+**Topic(s):**
+
+- General CAB process
+- SIP-015
+
+## Meeting Notes
+
+- The change proposed in SIP-015 to count mining creates a *potential* economic risk and attack vector
+  - Specifically the change: "...the median Bitcoin spend of the miner's last six block-commits will include late block-commits."
+  - Depending on how this is implemented, a user could potentially 'spoof' having been *attempting* to mine a majority of the previous 6 blocks
+  - By selectively doing this only when mining happens to be profitable, or when the miners corresponding stacking slot is up for rewards miners can abuse system
+
+- The changes proposed in SIP-015 to stacking methods (namely enabling contintual renewal and increasing of stacking) are critical primitives
+  - As long as these changes are backwards compatible, this CAB sees no economic risks or externalities
+  - Next steps would be to enable liquid stacking by supporting the create of an 'xSTX' token, conceptually similar to 'stETH'. Each 1 xSTX token would represent 1 stacked STX toke that accrues stacking rewards
```

Comment:
This exists: https://github.com/jcnelson/stx-future

Reply:
Thanks, will distribute it to rest of the Economics CAB, I think there may still be many edge cases and complexities in practically enabling liquid staked STX that need to be worked out (such as how to handle BTC rewards, the discrete cycle unlocks making a bit less fungible than stETH, etc)